### ISSUE TITLE
Horizontal sliders appear only after scrolling down to the end of the page

### ISSUE THREAD

**Issue by hauserulrich** [kanboard/kanboard#2935]
### Expected behaviour

If I look at a board or a gantt chart which is wider and higher than the window, I would expect horizontal sliders at the bottom of the window.

### Actual behaviour

The horizontal slider appears only if I scroll down the window until I reach the bottom.

### Steps to reproduce

1. Create a board or a gantt chart with many cards so that they are higher than the window and enough time frame or columns to exceed the window
2. You will see no horizontal slider at the bottom of the window to navigate horizontally.

### Configuration

    Application version: 1.0.36
    PHP version: 5.6.29
    PHP SAPI: fpm-fcgi
    OS version: Linux 4.4.0-45-generic
    Database driver: sqlite
    Database version: 3.11.0
    Browser: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:50.0) Gecko/20100101 Firefox/50.0

**Comment by fguillot** [comment]
Unfortunately, the scrollbars are placed by the browser itself. I do not have the control on that.

**Comment by KyleBrandon** [comment]
Actually this is controllable via CSS.  Add the following to the Custom Stylesheet and you'll get a horizontal scroll bar for the board view:

`body {
    height: 100vh;
}

#board-container {
       height: 100%;
       overflow-x: initial;
}`

**Comment by hauserulrich** [comment]
> Actually this is controllable via CSS. Add the following to the Custom Stylesheet and you'll get a horizontal scroll bar for the board view:

Unfortunately this does not help as the scrollbar is still invisible because it is below the lower end of the screen, if there are enough tasks in one column to fill more than a screen. Se the attached partial 

![kanboard_slider](https://cloud.githubusercontent.com/assets/8415116/25280983/1c2a9068-26ac-11e7-82a9-04ebcbfa363a.png)

**Comment by KyleBrandon** [comment]
Perhaps I'm thinking of something different.  The above CSS shows a horizontal scroll bar on Chrome and Firefox for me.
<img width="980" alt="screen shot 2017-04-21 at 9 15 11 am" src="https://cloud.githubusercontent.com/assets/1613205/25281509/42d83e88-2673-11e7-9e7e-6af585006621.png">

**Comment by psy-q** [comment]
Appears to be a duplicate of #2832. The issue is still there. It might be nice to have an approach similar to Trello, where clicking in the background allows dragging the entire page horizontally and vertically.

**Comment by bugwatcher** [comment]
This issue has been closed automatically by a bot for inactivity. For more information, read the [FAQ](https://docs.kanboard.org/en/1.2.5/developer_guide/faq.html).
